tests/config/install.sh

```diff
@@ -41,6 +42,12 @@ ln -sf $SRC_PATH/strings_dictionary.xml $DEST_SERVER_PATH/
 ln -sf $SRC_PATH/decimals_dictionary.xml $DEST_SERVER_PATH/
 ln -sf $SRC_PATH/executable_dictionary.xml $DEST_SERVER_PATH/
 
+# For CI when it is runned via init service
+mkdir -p /var/lib/clickhouse/cores
```

Comment:
By the way, what is this `cores` dir and why do we need it? Maybe we can handle the paths the same way we do for `server.key` etc? To have less default dirs with configs.

Reply:
>By the way, what is this cores dir and why do we need it?

`/var/lib/clickhouse/cores` is the cwd of the `clickhouse-serv` process under sysv-init (that is used for start `clickhouse-server` in CI except for fasttest)

> Maybe we can handle the paths the same way we do for server.key etc? To have less default dirs with configs.

But `server.key` is installed directly into the `/etc/clickhouse-server`, while I want to run those tests locally, and by not using absolute path to the list in `top_level_domains_lists.xml` it works now

Reply:
So it's dependent on the `init` system? This looks somewhat fragile. Maybe we can default to something in the data directory? Like `user_file` and `format_schemas`, we could use `/var/lib/clickhouse/top_level_domains` or something. Then, to run the tests locally, we can add an override to `programs/server/config.d/path.xml`.

Reply:
>So it's dependent on the init system? This looks somewhat fragile.

Indeed.

>Maybe we can default to something in the data directory? Like user_file and format_schemas, we could use /var/lib/clickhouse/top_level_domains or something. Then, to run the tests locally, we can add an override to programs/server/config.d/path.xml.

Looks a little bit overkill for this task, but maybe you right.

Reply:
@akuzm what do you think about just using user_files for this?

Reply:
> @akuzm what do you think about just using user_files for this?

This means they can be overwritten by users through SQL queries, not sure if this is OK or not.

Reply:
>This means they can be overwritten by users through SQL queries, not sure if this is OK or not.

You mean something like `INTO OUTFILE '/path/to/user_files/list.dat'`? Yeah, this should not be allowed, ok, thanks

Reply:
But fasttest will still be different, since it uses another data path